src/api/client.ts

```diff
@@ -81,6 +86,39 @@ export function getNostoClient(): PromiseLike<NostoClient> {
         }
     })
 }
+/**
+ * @group Nosto Client
+ * @category Core
+ */
+type LogLevel = "error" | "warn" | "info" | "debug"
```

Comment:
move to src/api/client.ts ?

Reply:
But it is in src/api/client.ts or did I miss understood?